src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
There should be a BEM block `stars` element, so you can apply modifiers to it `stars--0`.
```suggestion
    <div class="stars stars--0">
```